dockerfiles/base/Dockerfile.ubi

```diff
@@ -54,9 +54,6 @@ RUN <<EOF
     rpm --erase --nodeps dnf-plugin-subscription-manager
     rpm --erase --nodeps gdb-gdbserver
     rpm --erase --nodeps glibc-common
-    rpm --erase --nodeps glibc-gconv-extra
-    rpm --erase --nodeps glibc-langpack-en
-    rpm --erase --nodeps glibc-minimal-langpack
```

Comment:
@aarlaud, why do we don't remove it? these deps contain high vulns iirc.

Reply:
It fails on these saying they are not existent. Maybe the base image changed and they were removed?

Reply:
ok, I'll investigate it later. ✅  for this one.